flake.nix

```diff
@@ -261,7 +263,20 @@
             "-${client.version}-against-${daemon.version}";
         inherit version;
 
-        src = nixSrc;
+        src = fileset.toSource {
+          root = ./.;
+          fileset = fileset.intersect baseFiles (fileset.unions ([
+            ./.version
+            ./configure.ac
+            ./local.mk
+            ./m4
+            ./Makefile
+            ./Makefile.config.in
+            ./mk
+            # TODO: do we really need README.md? It doesn't seem used in the build.
+            ./README.md
```

Comment:
These are duplicated with `nixSrc`. Call them `coreFiles`?

Funnily `baseFiles` confused me for a while. Maybe that should be `upperBoundFiles` or something.